### ISSUE TITLE
Unable to load old particle effect, unable to locate image

### ISSUE THREAD

**Issue by dkoding** [libgdx/gdx-particle-editor#5]
I have a particle like this:

```
...the rest is omitted for brevity...
- Options - 
attached: false
continuous: false
aligned: false
additive: false
behind: false
premultipliedAlpha: false
spriteMode: single
- Image Paths -
particleblack.png
```

The particleblack.png is located at a different location so loading fails. I get a dialog asking me to find the particle. After locating it and clicking 'Load Particle Effect' it still fails to load the particle, and then freezes. Only thing I can do is to close the window to exit (the [x] in top right corner)

It works correctly in the old particle editor (I still have to locate the particle)

**Comment by raeleus** [comment]
This should have been addressed in update 1.0.3. Is that the version you're using? If you are, can you send me the full file for me to test? Thanks.

**Comment by raeleus** [comment]
Closing due to inactivity.
